pages/linux/htpdate.md

```diff
@@ -0,0 +1,13 @@
+# htpdate
+
+> Synchronize local date and time via HTTP headers from web servers.
+> Requires administrator privileges.
+> More information: <http://www.rkeene.org/oss/htp/>.
+
+- Synchronize date and time:
+
+`sudo htpdate {{host}}`
```

Comment:
We don't use `sudo` in the pages.
```suggestion
`htpdate {{host}}`
```

Reply:
Erm @navarroaxel, that's not quite true. We always prefix a command with `sudo` if root privileges are required in order to execute said command.